src/commands/createNewProject/ProgrammingModelStep.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureWizardPromptStep, IAzureQuickPickItem, IAzureQuickPickOptions, nonNullValue, openUrl } from '@microsoft/vscode-azext-utils';
-import { learnMoreQp, nodeProgrammingModelSetting } from '../../constants';
```

Comment:
Is the `learnMoreQp` constant still referenced by anything else or can it be removed?

Reply:
Still used, but only in one other place. I'll just move it to there.